src/dashboard/src/media/css/archival_storage.css

```diff
@@ -124,3 +124,17 @@ div.search-summary p {
 #create-aic-btn {
   margin-bottom: 10px;
 }
+
+#download-csv-btn {
+  float: right;
+  border: none;
+  background: none;
+  margin-top: 10px;
+}
+
+#unused {
+  border: none;
+  background: none;
+  margin-bottom: 0px;
+  padding-bottom: 0px;
+}
```

Comment:
Is this rule set leftover from the development process?

Reply:
:facepalm: doh! yes! Good catch. Will remove @tw4l.

Reply:
Sounds good, thanks! :smile: